envoy.yaml

```diff
@@ -63,5 +63,23 @@ pipeline:
       bazel build --verbose_failures -c opt envoy
 
       mkdir -p ${{targets.destdir}}/usr/bin/
-      cp bazel-bin/source/exe/envoy-static ${{targets.destdir}}/usr/bin/
+      mv bazel-bin/source/exe/envoy-static ${{targets.destdir}}/usr/bin/envoy
   - uses: strip
+
+subpackages:
+  - name: envoy-oci-entrypoint
+    description: Entrypoint for using Envoy in OCI containers
+    dependencies:
+      runtime:
+        - busybox
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/var/lib/envoy/init
+          cp envoy-entrypoint.sh ${{targets.subpkgdir}}/var/lib/envoy/init/
```

Comment:
is this part of envoy sources?

Reply:
great point and catch! yes it is :) will update the PR

Reply:
I'd missed the `git add --all` to add the files in the `./envoy` source-dir in the previous commit, in case it was a bit confusing.  I've switched the references to used the files from the fetched source instead.